Source/WebKit/WebProcess/WebPage/wc/WCUpdateInfo.h

```diff
@@ -45,7 +45,7 @@ struct WCTileUpdate {
     WebCore::IntRect dirtyRect;
 };
 
-enum class WCLayerChange : uint32_t {
+enum class WCLayerChange : uint8_t {
     Children                = 1 <<  0,
```

Comment:
Revert this change and mark the serialization files definition as uint32_t

This is why compilation is failing